<h1 align="center">Olga Bagrova</h1>
</div>

<p align="center">
<b>‚öõÔ∏èPhysics in my mind üß† üß¨Biology in my heart‚ù§Ô∏è üíªCoding in my soulüïäÔ∏è</b>
</p>

[![LinkedIn](https://img.shields.io/badge/olgabagrova-%230077B5?style=flat-square&logo=linkedin&logoColor=white)](https://linkedin.com/in/olga-bagrova-5812422a0)
[![Gmail](https://img.shields.io/badge/-olgabagrova98@gmail.com-c14438?style=flat-square&logo=Gmail&logoColor=white)](mailto:olgabagrova98@gmail.com) 
[![Telegram](https://img.shields.io/badge/-@OlgaB_MSU-%232CA5E0?style=flat-square&logo=Telegram&logoColor=white)](https://t.me/OlgaB_MSU)
<!---[![Twitter](https://img.shields.io/badge/Twitter--alga_3-brightgreen?style=social&logo=Twitter)](https://twitter.com/alga_3)-->


<p>
This is the github of an aspiring bioinformatician. I graduated from the <a href="https://bioinf.me/en">Bioinformatics Institute</a> as a biostatistician and bioinformatician and the <a href="https://msu.ru/en/">Lomonosov Moscow State University</a> with BSc and MSc degrees in Biophysics. My experience encompasses collaboration with virologists, biochemists, and immunologists. However, my primary passion lies in researching the structure and evolution of proteins.
</p>



## Skills:
![Python](https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)
![Pandas](https://img.shields.io/badge/pandas-%23150458.svg?style=for-the-badge&logo=pandas&logoColor=white) 
![NumPy](https://img.shields.io/badge/numpy-%23013243.svg?style=for-the-badge&logo=numpy&logoColor=white) 
![Matplotlib](https://img.shields.io/badge/Matplotlib-%23ffffff.svg?style=for-the-badge&logo=Matplotlib&logoColor=black)
![SciPy](https://img.shields.io/badge/SciPy-%230C55A5.svg?style=for-the-badge&logo=scipy&logoColor=%white)
![scikit-learn](https://img.shields.io/badge/scikit--learn-%23F7931E.svg?style=for-the-badge&logo=scikit-learn&logoColor=white) 
![PyTorch](https://img.shields.io/badge/PyTorch-%23EE4C2C.svg?style=for-the-badge&logo=PyTorch&logoColor=white)
<!--![Biopython](https://svgshare.com/i/189J.svg)--><!---https://github.com/Olga-Bagrova/readme_checking/blob/main/badges/biopython.svg?raw=true-->
![R](https://img.shields.io/badge/r-%23276DC3.svg?style=for-the-badge&logo=r&logoColor=white) 
![Tidyverse](https://img.shields.io/badge/Tidyverse-161042?style=for-the-badge&logo=Tidyverse&logoColor=white) <!---dplyr, ggplot2, tidyverse, tidyr, ape, ggtree-->
![Linux](https://img.shields.io/badge/Linux-FCC624?style=for-the-badge&logo=linux&logoColor=black)
![Git](https://img.shields.io/badge/git-%23F05033.svg?style=for-the-badge&logo=git&logoColor=white)
![Anaconda](https://img.shields.io/badge/Anaconda-44A833?style=for-the-badge&logo=anaconda&color=525252)
![GNU Bash](https://img.shields.io/badge/GNU%20Bash-000000?style=for-the-badge&logo=GNU%20Bash&logoColor=white)

<details><summary>
<b>Research experience</b>
</summary><br> 
  
### Studying population frequencies of T-cell receptor (TCR) alleles using immune repertoire sequencing
<!--Publication/Results-->
[Repo](https://github.com/Olga-Bagrova/RepSeq_TCRanalysis)

*Comprehensive analysis of the TCR repertoire for a large group of donors.*

We analyzed:
* Gene usage distribution of the TRA and TRB chains for the identification of deletions
* Co-expression factors for V-V, J-J, V-J pairs within and between TRA and TRB chains 
* Genes usage between only functional and non-functional sequences to identify thymus selection
* We also compared found patterns between populations

* **Skills:** Python (*scipy, statsmodels, numpy, pandas, matplotlib, seaborn, re, os*), Bash, Jupyterhub, Conda, Biological databases (*IMGT*). <br>

  

### The analysis of the secondary structures distributions along the polypeptide chains of proteins within different functional classes, homologous proteins and topologous proteins
<!--Results. -->
[Repo](https://github.com/Olga-Bagrova/DistrProtStruc)

*Development of a new representation of proteins for comparing structures with a focus on the secondary structures distribution.*

* New representation of protein molecules using the distributions of secondary structures along their chains was developed
* Proteins from PDB were divided into groups according to function and homology
* Frequencies of occurrence of various secondary structures for the selected groups were compared

* **Skills:** Python (*biotite, numpy, pandas, matplotlib, seaborn, re, os*), Bash, Conda, Biological databases (*PDB, CATH, UniProt, NCBI, PFam, GO*). <br>



### Biostatistical analysis of melanoma patients‚Äô transcriptomic data from open database TCGA
<!--Results. -->
[Repo](https://github.com/Olga-Bagrova/TCGA_SKCM)

*Analyzing transcriptomic data from melanoma samples to identify trends in gene expression and building a model to predict overall survival based on expression levels.*

* The study of clinical and transcriptomic data of patients with melanoma to identify patient groups and their patterns
* Transcriptomic signatures were selected based on a literature review and Cox regression to predict overall patient survival
* A survival prediction model using Cox regression has been developed

* **Skills:** R (*survival, survminer, glmnet, ComplexHeatmap, tidyverse, gtsummary, factoextra, ggbiplot, ggplot2, ggpubr, dplyr, plotly, tibble, matrixStats*), Biological databases (*TCGA, GO*).<br>

</details>

